test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.LogProperties.cs

```diff
@@ -619,36 +618,12 @@ class MyClass
             internal static partial class C
             {
                 [LogMethod(0, LogLevel.Debug, ""Only {A}"")]
-                static partial void M(ILogger logger, IRedactorProvider provider, [FeedbackData] string a, [LogProperties] MyClass param);
```

Comment:
As @klauco pointed, we should still support this use-case. Maybe not in 3P though 🤔

Reply:
Removing redactor provider support was a huge simplification. I'd really need to be convinced that supporting it is a useful thing within the redaction infra.

Note that there's nothing preventing the customer from grabbing an IRedactorProvider themselves and redacting stuff on their own before calling the lower-level log method.

Reply:
can we provide analyzer and fixer for existing customers, to move to the signature which does not use IRedactorProvider?